nbdev/release.py

```diff
@@ -236,6 +236,15 @@ def write_conda_meta(path='conda'):
     _write_yaml(path, *_get_conda_meta())
 
 # %% ../nbs/api/release.ipynb 43
+def write_requirements(directory=None):
+    "Writes a `requirements.txt` file to `directory` based on settings.ini."
+    cfg = get_config()
+    d = cfg.config_path if not directory else Path(directory)
```

Comment:
Extremely minor, but you don't need to `not` there

```suggestion
    d = Path(directory) if directory else cfg.config_path
```